submissions/YuraZagor/hooliPopup/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>myPopup</title>
+</head>
+<body>
+	<nav>
+		<ul class="ul--menu">
+			<li class="li--menu"><a href="#">Mail</a></li>
+			<li class="li--menu"><a href="#">Images</a></li>
+
+			<li class="li--menu popup pic" >
+				<label id="popup-label" class="sub-menu" for='checkbox-icon'> <img src="./popup-icons/popup-button.png" alt="popup-label" > </label>
+				<input type="checkbox" id="checkbox-icon" name='checkbox-icon' class='checkbox-icon' alt="popup-checkbox" >
+
+				<section class="outer-popup-container">
+					<div class="inner-container">
+						<ul class="popups--container">
+							<li class="li--popup--item"> <a href="#"><img src="./popup-icons/blogs.png" alt="articles" class="popup--item"> </a></li>
```

Comment:
Please fix styles on focused state [Watch video](https://share.vidyard.com/watch/cJKtB6pUbCL6JqXeP6gZif?)

Reply:
Still not fixed :cry: 
![image](https://user-images.githubusercontent.com/16196199/187144604-33657283-b1f1-4c27-bb42-872b44a491aa.png)

Reply:
Sorry, was completely stuck on menu.   Seems now it's working okay.  As I got it empty hrefs  still had their reserved space  and their container needed tuning.